### PR TITLE
Document the 90 day pause/restore window for free tier

### PR DESCRIPTION
**Pull request by olirice**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?
Documents that free tier projects have 90 days to restore on the platform and then the restore button becomes a "download backup" button

### CODE REVIEW

**Review by charislam — APPROVED** [review]
Made some edits, you might want to take a look to make sure I haven't twisted your words, but otherwise it's good to go!

File: apps/docs/content/guides/platform/pause-restore.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+id: 'pause-restore'
```
Comment (darora) [inline apps/docs/content/guides/platform/pause-restore.mdx:2]: Are we deleting https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#pause--restore ? Or linking back and forth between the two? Unclear what should go on this page vs the other one

File: apps/docs/content/guides/platform/pause-restore.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+title: 'Pausing and Restoring'
+description: 'Pausing and restoring projects.'
+---
+
+Organizations on a Free Plan can [pause](/dashboard/project/_/settings/general) and restore projects. This can be done to conserve resources for unused projects, or to upgrade a project to the latest version of each service.
+
+<Admonition type="note">
+
+Free projects are automatically paused after <SharedData data="config">pausing.free_tier</SharedData> of inactivity.
```
Comment (darora) [inline apps/docs/content/guides/platform/pause-restore.mdx:11]: "Starting on 13th June" (or similar)

File: apps/docs/content/guides/platform/pause-restore.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### 90-day restore window
+
+During the 90-day restore window, you can restore a paused project from [your project dashboard](/dashboard/projects).
+
+<Image alt="Project Paused: 90 Days Remaining" src="/docs/img/guides/platform/paused-90-day.png" />
+
+### Post-restore window
+
+After the 90-day restore window, you can download your project's backup file from the project dashboard. This backup is compatible with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html). You can extract your data from it, or attempt to manually restore the project directly.
```
Comment (darora) [inline apps/docs/content/guides/platform/pause-restore.mdx:29]: Mention that pgsodium-managed data isn't handled?

**Review by charislam — APPROVED** [review]
Tested build: opened preview

File: apps/docs/content/guides/platform/pause-restore.mdx (hunk 3 above)
Comment (olirice) [reply apps/docs/content/guides/platform/pause-restore.mdx:29]: Wouldn't they receive the encrypted data and then hit the management api to retrieve their encryption key?

Even if its a multi-file backup we should still provide it to them and provide minimal instructions on how to restore using it

Comment (olirice) [reply apps/docs/content/guides/platform/pause-restore.mdx:29]: I'm going to merge without this change to unblock everything else but will create a separate PR to update the docs with your feedback

File: apps/docs/content/guides/platform/pause-restore.mdx (hunk 1 above)
Comment (olirice) [reply apps/docs/content/guides/platform/pause-restore.mdx:2]: I deleted the new page and added the content under a `Time Limits` header of the existing `Caveats` section in `migrating-and-upgrading`

File: apps/docs/content/guides/platform/pause-restore.mdx (hunk 3 above)
Comment (darora) [reply apps/docs/content/guides/platform/pause-restore.mdx]: Yeah I think the part that's missing right now is more detailed instructions (e.g. a guide) on how they can restore the backup that they download.
Pretty sure right now the link is for a single-file backup, which would double-encrypt on restore (unless we have instructions to edit it to disable triggers etc, quite an involved process)

**Pull request merged**